### ISSUE TITLE
Shuffling of the data

### ISSUE THREAD

**Issue by FAhtisham** [FernandoLpz/Text-Classification-LSTMs-PyTorch#3]
Hello, I am sorry if my question sounds stupid to you, i have gone through your example but i was unable to spot the shuffling of the data in your code. Can you please shed some light on it ?

**Comment by FernandoLpz** [comment]
Hello @FAhtisham, 

The shuffling is done when splitting data with the scikit-learn function, such as:
`self.x_train, self.x_test, self.y_train, self.y_test = train_test_split(X, Y, test_size=self.test_size)`
This `train_test_split` function has a parameter by default which is `shuffle`. You can take a look at:
https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html

**Comment by FAhtisham** [comment]
Hi @FernandoLpz , thanks
Your code is working with this suffling via scikit. The problem occurs when we use shuffle in pytorch DataLoader which results in the same Training and Testing accuracy (0.49, 0.50). Do you think DataLoader shuffle works differently ? or I am committing a mistake by shuffling DataLoader.

PS: The tutorial itself on medium was very precise and explanatory.
